### DISCUSSION TITLE
GUI for zfs

### DISCUSSION THREAD

**Discussion by isZYKerman** [openzfs/zfs#16711] [Q&A]
I migrated my NAS from TrueNAS to Ubuntu recently. I want a nice GUI to manage all the scrub, replication and other  tasks just as convenient as in TrueNAS. Are there any free options?

**Comment by amotin** [comment]
TrueNAS? ;)

**Reply by isZYKerman** [reply]
But I've just migrated from TrueNAS to ubuntu🧐🧐🧐

**Comment by guenther-alka** [comment]
napp-it cs, free for noncommercial use (Port from the Illumos/Solaris edition)

The Web-Gui runs on Windows where it can manage Storage Spaces and the upcoming OpenZFS. It can remotely manage ZFS servers on BSD, OSX, Linux, Illumos or Windows with replication any to any, zfs and snap management, encryption with 2/3way keysplit on remote https servers and autounlock on boot. 

https://www.napp-it.org/manuals/index_en.html

**Reply by isZYKerman** [reply]
Thanks. I‘ll try it out
